indy_client/test/cli/test_pool_upgrade.py

```diff
@@ -100,9 +100,8 @@ def test_force_false_upgrade(
 
```

Comment:
Please add a test where package name is not specified (a default value should be used)

Reply:
there are several tests which use send_upgrade_cmd func - it uses default for package parameter